File: GameEngine_IOS/GameEngine/Commons/ColorRGB.swift
import Foundation

/**
 * Represents one color with (Red, Green and Blue) components
 */
public class ColorRGB {
    
    /**
     * Red component
     */
    var r: Float;
    
    /**
     * Green component
     */
    var g: Float;
    
    /**
     * Blue component
     */
    var b: Float;
    
    /**
     * @param r
     *            Red component
     * @param g
     *            Green component
     * @param b
     *            Blue component
     */
    public init(r : Float, g : Float, b: Float) {
        self.r = r;
        self.g = g;
        self.b = b;
    }
    
}

File: GameEngine_Android/commons/src/main/java/com/dferreira/commons/ColorRGB.java
package com.dferreira.commons;

/**
 * Represents one color with (Red, Green and Blue) components
 */
public class ColorRGB {

    /**
     * Red component
     */
    public final float r;

    /**
     * Green component
     */
    public final float g;

    /**
     * Blue component
     */
    public final float b;

    /**
     * @param r Red component
     * @param g Green component
     * @param b Blue component
     */
    public ColorRGB(float r, float g, float b) {
        super();
        this.r = r;
        this.g = g;
        this.b = b;
    }

}